libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/areaChart/tests/PluggableAreaChart.test.tsx

```diff
@@ -797,4 +801,18 @@ describe("PluggableAreaChart", () => {
             expect(sortConfig).toMatchSnapshot();
         });
     });
+
+    describe("`renderVisualization` and `renderConfigurationPanel`", () => {
```

Comment:
I would also see it as an anti-pattern: it might muddle error messages in case a test fails, etc. This particular case looks like it should be better handled by some parametric test? So for each vis type it does this?

Reply:
Yes, that would probably be a cleaner approach. I'll try that the next time I'm touching this code, if you don't mind